dev-tools/scripts/releaseWizard.yaml

```diff
@@ -1067,6 +1067,70 @@ groups:
         cmd: svn -m "Add docs, changes and javadocs for Solr {{ release_version }}"  import {{ git_checkout_folder }}/solr/build/docs  https://svn.apache.org/repos/infra/sites/solr/docs/{{ version }}
         logfile: add-docs-solr.log
         comment: Add docs for Solr
+  - !Todo
+    id: build_final_ref_guide
+    title: Build the ref guide
+    links:
+      - https://solr.apache.org/guide/{{ release_version_refguide }}
+    commands: !Commands
+      root_folder: '{{ git_checkout_folder }}'
+      commands_text: Build the Solr Ref Guide
+      commands:
+        - !Command
+          cmd: "{{ gradle_cmd }} solr:solr-ref-guide:clean"
+          comment: Clean the Ref Guide build
+        - !Command
+          cmd: "{{ gradle_cmd }} solr:solr-ref-guide:buildSite -PsolrGuideDraft=false"
+          comment: Build Ref Guide
+  - !Todo
+    id: upload_final_ref_guide
+    title: Upload the ref guide
+    links:
+      - https://solr.apache.org/guide/{{ release_version_refguide }}
+    depends:
+      - build_final_ref_guide
+    types:
+      - major
+      - minor
+    commands: !Commands
+      root_folder: '{{ git_checkout_folder }}'
+      commands_text: Upload the Solr Ref Guide to SVN production tree
+      commands:
+        - !Command
+          cmd: svn -m "Add Solr Ref Guide {{ release_version }}" import {{ git_checkout_folder }}/solr/solr-ref-guide/build/html-site  https://svn.apache.org/repos/infra/sites/solr/guide/{{ release_version_refguide }}
+          logfile: add-docs-solr.log
+          comment: Upload Solr Ref Guide
+  - !Todo
+    id: upload_final_ref_guide
+    title: Upload (and overwrite-write) the ref guide
+    links:
+      - https://solr.apache.org/guide/{{ release_version_refguide }}
+    depends:
+      - build_final_ref_guide
+    types:
+      - patch
+    commands: !Commands
+      root_folder: '{{ ref_guide_svn_folder }}'
+      commands_text: Upload the Solr Ref Guide to SVN production tree, overwriting the previous version
+      commands:
+        - !Command
+          cmd: svn co https://svn.apache.org/repos/infra/sites/solr/guide/{{ release_version_refguide }} "{{ ref_guide_svn_folder }}"
+          logfile: ref-guide-svn-co.log
+          comment: Checkout the existing Solr Ref Guide for version {{ ref_guide_svn_folder }}
+          root_folder: '{{ git_checkout_folder }}'
+        - !Command
+          cmd: cp -r {{ git_checkout_folder }}/solr/solr-ref-guide/build/html-site {{ ref_guide_svn_folder }}
+          comment: Copy and overwrite new Solr Ref Guide
+          root_folder: '{{ git_checkout_folder }}'
+        - !Command
+          cmd: svn status | grep '^!' | awk '{print $2}' | xargs svn delete
```

Comment:
Will this prompt for confirmation? I'm a little nervous about running a blind svn delete

Reply:
Nah, but definitely a good idea to add that before the commit